Common/Securities/MarketHoursDatabase.cs

```diff
@@ -98,20 +98,6 @@ public SecurityExchangeHours GetExchangeHours(string market, Symbol symbol, Secu
             return GetEntry(market, symbol, securityType, overrideTimeZone).ExchangeHours;
         }
 
-        /// <summary>
-        /// Performs a lookup using the specified information and returns the data's time zone if found,
-        /// if an entry is not found, an exception is thrown
-        /// </summary>
-        /// <param name="market">The market the exchange resides in, i.e, 'usa', 'fxcm', ect...</param>
-        /// <param name="symbol">The particular symbol being traded</param>
-        /// <param name="securityType">The security type of the symbol</param>
-        /// <returns>The raw data time zone for the specified security</returns>
-        public DateTimeZone GetDataTimeZone(string market, Symbol symbol, SecurityType securityType)
```

Comment:
The reasoning behind removing this was to force the caller to think about the override time zone. I'm beginning to think that a better route is to add an entry to market hours database when we do `AddData<T>` -- this will ensure the entire system gets the right value without needing to concern itself about custom data and overriden time zones.

A side effect s that we would remove the concept of overriding a time zone; replaced by entering a symbol specific entry to define the override.